projects/ssr-tests/src/utils/ssr.utils.ts

```diff
@@ -51,7 +51,7 @@ export async function startSsrServer({
   timeout = DEFAULT_SSR_TIMEOUT,
 }: SsrServerOptions = {}) {
   child = childProcess.spawn(
-    `NODE_TLS_REJECT_UNAUTHORIZED=0 SSR_CACHE=${cache} SSR_TIMEOUT=${timeout} PORT=${port} npm run serve:ssr --prefix ../../> .ssr.log`,
+    `NODE_TLS_REJECT_UNAUTHORIZED=0 SSR_CACHE=${cache} SSR_TIMEOUT=${timeout} PORT=${port} npm run serve:ssr --prefix ../../> .ssr.log 2>&1`,
```

Comment:
`minor:` What do you think about adding a description of what exactly `2>$1` does? It would be helpful to leave a reference for the future.